Library/Homebrew/sandbox.rb

```diff
@@ -32,7 +32,13 @@ def initialize
   end
 
   def record_log(file)
-    @log = file
+    @logfile = file
+  end
+
+  def self.log_warning_message; <<-EOS.undent
+      We use time to filter sandbox log. Therefore, unrelated logs may be recorded in below.
+
+    EOS
   end
```

Comment:
How about only writing it to log file? Usually there won't be unrelated logs for bot. But on users computer, the chance is high which can cause confusion.